AppVeyor batch and Coveralls token

kevcunnane opened the pull request.

msftclas commented:
Hi __@kevcunnane__, I'm your friendly neighborhood Microsoft Pull Request Bot (You can call me MSBOT). Thanks for your contribution!
    <p>
        It looks like you're a Microsoft contributor (Kevin Cunnane). If you're full-time, we DON'T require a Contribution License Agreement. If you are a vendor, please DO sign the electronic Contribution License Agreement. It will take 2 minutes and there's no faxing! https://cla.microsoft.com.
    </p>

TTYL, MSBOT;

anthonydresser commented on README.md:
```diff
@@ -1,5 +1,5 @@
 [![Travis CI](https://travis-ci.org/Microsoft/sqltoolsservice.svg?branch=dev)](https://travis-ci.org/Microsoft/sqltoolsservice)
-[![AppVeyor](https://ci.appveyor.com/api/projects/status/github/{INSERTUSERHERE}/sqltoolsservice?svg=true&retina=true&branch=dev)](https://ci.appveyor.com/project/{INSERTUSERHERE}/sqltoolsservice)
+[![AppVeyor](https://ci.appveyor.com/api/projects/status/g48acrfwbx7pj452?svg=true)](https://ci.appveyor.com/project/kburtram/sqltoolsservice)
```
You don't need to refer to it with this token, you can be explicit since we are using an open source repo. See https://www.appveyor.com/docs/status-badges/#badges-for-projects-with-public-repositories-on-github-and-bitbucket

kevcunnane replied:
This was failing for me with permissions denied issue. Any ideas why? Message:
`{"message":"Project not found or access denied."}`

kevcunnane replied:
Thanks! Will fix

anthonydresser commented:
It looks like PR code coverage is a little more complicated on Appveyor. See https://dzone.com/articles/code-coverage-on-github-prs-with-coverallsio-1

kevcunnane commented:
@anthonydresser I've got AppVeyor working with Coveralls on PRs now, will add in the webhook for Coveralls after this checkin. I'm going to commit as it's all working OK and I think I've addressed all the issues
